### PR TITLE
Update SentinelOne documentation

### PR DESCRIPTION
**Pull request by TonioRyo**
* Deprecate SentinelOne Deep Visibility integration
* Ability to use Service Users in SentinelOne management console

### CODE REVIEW

File: docs/xdr/features/collect/integrations/endpoint/sentinelone_deepvisibility.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 uuid: 8c1bee36-d516-42f9-9b6f-a8e4dcac3d1d
-name: SentinelOne Deep Visibility
+name: SentinelOne Deep Visibility [DEPRECATED]
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/endpoint/sentinelone_deepvisibility.md:2]: The name is not synced with this intake-format repository

https://github.com/SEKOIA-IO/intake-formats/commit/990e739f7dc1f6ca759c686dbfa3bb7ad61f35e1#diff-24e16cf9f977119509f99a9de017e9f90b1dce102773faa718d77c35df5e977fR2

**Review by squioc — APPROVED** [review]

File: docs/xdr/features/collect/integrations/endpoint/sentinelone.md [hunk 2]
```diff
@@ -18,28 +18,26 @@ And depending on the context of the log, additional content could be available,
 - Network information
 - File information
 
-> For advanced log collection, we suggest you to use SentinelOne Deep Visibility kafka option, as described offered by the [SentinelOne DeepVisibility integration](sentinelone_deepvisibility.md).
+> For advanced log collection, we suggest you to use SentinelOne Cloud Funnel 2.0 option, as described offered by the [SentinelOne Cloud Funnel 2.0 integration](sentinelone_cloudfunnel2.0.md).
```
Comment (ka0ula) [inline docs/xdr/features/collect/integrations/endpoint/sentinelone.md:21]:
```suggestion
!!! Tip
    For advanced log collection, we suggest you to use SentinelOne Cloud Funnel 2.0 option, as described offered by the [SentinelOne Cloud Funnel 2.0 integration](sentinelone_cloudfunnel2.0.md).
     
```

File: docs/xdr/features/collect/integrations/endpoint/sentinelone.md [hunk 3]
```diff
@@ -18,28 +18,26 @@ And depending on the context of the log, additional content could be available,
...
 
 ## Configure
 
-This setup guide will show you how to pull events produced by SentinelOne EDR on [SEKOIA.IO](https://app.sekoia.io/). To collect the SentinelOne logs, you must generate an API token from the SentinelOne Management Console
+This setup guide will show you how to pull events produced by SentinelOne EDR on [SEKOIA.IO](https://app.sekoia.io/). To collect the SentinelOne logs, you must generate an API token from the SentinelOne Management Console. We recommend creating a Service User to use a dedicated account for the integration.
 
 **Important**: If you have multiple SentinelOne Management Consoles, you must generate an API Token for each one.
 
-> The API token you generate is time limited. To regenerate a new token (and invalidate the old one), log in with the dedicated SentinelOne account. You do not need to create a new account.
+> The API token you generate is time limited. To regenerate a new token (and invalidate the old one), you will need to copy the Service User. Please refer to the SentinelOne documentation to obtain guidance on how to do this action.
```
Comment (ka0ula) [inline docs/xdr/features/collect/integrations/endpoint/sentinelone.md:33]:
```suggestion
!!! note
    The API token you generate is time limited. To regenerate a new token (and invalidate the old one), you will need to copy the Service User. Please refer to the SentinelOne documentation to obtain guidance on how to do this action.
```

File: docs/xdr/features/collect/integrations/endpoint/sentinelone.md [hunk 4]
```diff
@@ -18,28 +18,26 @@ And depending on the context of the log, additional content could be available,
...
+2. Create a new `Service User` by specifying a name and an expiration date.
+3. Choose the `Scope` of the `Service User`: `Global`, `Account` or `Site`, select the appropriate `Account(s)` or `Site(s)` and the role to grant to the `Service User`
+4. Click on `Create User` and copy the API token generated.
 
-**Note**: A user with a role of "Site Admin" can mitigate threats from the [SEKOIA.IO](https://app.sekoia.io/). A user with a role of "Site Viewer" can view threats but cannot take action.
-**Note**: You can generate a token only for your own user.
-
-3. Click `Generate` next to API Token. It shows the token string and the date that the token expires.
-4. Copy the token or click `Download` to save it.
+**Note**: A `Service User` with a role of `Site Admin` or `IR Team` can mitigate threats from [Sekoia.io](https://app.sekoia.io/) using [SentinelOne playbook actions](../../../automate/library/sentinel-one.md). A user with a role of `Site Viewer` can view activity events and threats but cannot take action.
```
Comment (ka0ula) [inline docs/xdr/features/collect/integrations/endpoint/sentinelone.md:40]:
```suggestion
!!! note
    A `Service User` with a role of `Site Admin` or `IR Team` can mitigate threats from [Sekoia.io](https://app.sekoia.io/) using [SentinelOne playbook actions](../../../automate/library/sentinel-one.md). A user with a role of `Site Viewer` can view activity events and threats but cannot take action.
```

**Review by ka0ula — CHANGES_REQUESTED** [review]

**Pull request merged**